index.d.ts

```diff
@@ -23,7 +23,7 @@ declare class Emittery {
 
 	@returns An unsubscribe method.
 	*/
-	on(eventName: string, listener: (eventData?: unknown) => void): Emittery.UnsubscribeFn;
+	on(eventName: (string | symbol), listener: (eventData?: unknown) => void): Emittery.UnsubscribeFn;
```

Comment:
We should probably move the type for `eventName` to a separate typing since it's used throughout the types.